app/tree/app/graphql/GraphQlController.scala

```diff
@@ -0,0 +1,12 @@
+package graphql
+
+import akka.actor.ActorSystem
+import io.github.tetherlessworld.twxplore.lib.base.controllers.graphql.AbstractGraphQlController
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.Request
+import stores.Store
+
+@Singleton
```

Comment:
GraphQL stuff should be in a separate/later commit if this is about parse file changes. Will let this slide for now, but in the future please group logical changes in commits.